Dockerfile

```diff
@@ -227,14 +252,14 @@ RUN apt-get update -qq && \
         build-essential \
         libffi-dev \
         libssl-dev \
-        python3.9 \
+        python3 \
         python3-dev \
         python3-pip && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.9 1
-ENV PYTHON_VERSION=3
+ENV PYTHON_VERSION=3 \
+    PIP_BREAK_SYSTEM_PACKAGES=1
```

Comment:
It would rather prefer to install the packages for the user (`--user`) or within a virtual environment. Although virtual environment seems a bit excessive to me.

Reply:
This has been addressed using virtual environments.